rfcs/proposed/zip_view/README.md

```diff
@@ -0,0 +1,40 @@
+# zip_view Support for the oneDPL Range APIs with C++20
+
+## Introduction
+`std::ranges::zip_view` is a powerful utility that enables developers to combine two or more ranges into a single view,
+where each element is represented as a tuple containing corresponding elements from each input range.
+
+## Motivations
+`std::ranges::zip_view` is a convenient way to combine multiple ranges into a single view, where each element of
+the resulting range is a tuple containing one element from each of the input ranges. This can be particularly
+useful for iterating over multiple collections in parallel. `std::ranges::zip_view` was introduced in C++23,
+but many developers are still using C++20 standard. So, oneDPL introduces `oneapi::dpl::ranges::zip_view`,
+with the same API and functionality as `std::ranges::zip_view`.
+
+### Key Requirements
+`oneapi::dpl::ranges::zip_view` should be:
+- compilable with C++20 version (minimum)
+- API-compliant with `std::ranges::zip_view`
+- random accessible view; the "underlying" views also should be random accessible
+- in case of a device usage: a device copyable view itself and the "underlying" views also should be device copyable
+  
+`oneapi::dpl::ranges::zip_view::iterator` should be:
+- value-swappable (https://en.cppreference.com/w/cpp/named_req/ValueSwappable)
+- implicitly convertible to `oneapi::dpl::zip_iterator`
```

Comment:
Another compatibility/conversion question is with regard to the C++23 `zip_view`. If our view cannot always be used in place of the standard one, perhaps we should consider how developers would switch between the two.

Reply:
Do you mean the standard `zip_view` C++23 might be not device copyable f.e.? And how developers can use our view instead of?

Reply:
No, I meant if our view is not going to support all requirements/use cases of the standard (e.g. not going to work with bidirectional etc. views), then developers cannot use it all the times. So it makes sense to think if there could be cases to transit from one zip view to another.